tcms/wsgi.py

```diff
@@ -37,15 +37,15 @@
 # This application object is used by any WSGI server configured to use this
 # file. This includes Django's development server, if the WSGI_APPLICATION
 # setting points here.
-_application = get_wsgi_application()
```

Comment:
I have a pale memory, that with @atodorov agreed that we will not rename that, because we are not sure who is calling this and how, and renaming it might break something.

Reply:
That was back then, but looking at it I don't think it will actually break something. We do have a docker-build step in the test suite, what is missing though is firing up the application and making sure it works. 

I vote for committing this and if necessary we can revert the commit.